platform/templates/secrets/guardian.yaml

```diff
@@ -1,9 +1,9 @@
-{{- if .Values.ldap.enabled }}
+{{- if .Values.openldap.enabled }}
 apiVersion: v1
 kind: Secret
 metadata:
   name: guardian
 type: Opaque
 data:
-  secret_key: bG9jYWxob3N0Cg== #localhost
-{{- end }}
+  secret_key: ce1RV/hBmrFVh11geYV4wPWvZ9vOF1en/tZTD/Qwlr27UmJka+Qi+jbFAZLu7FAu
```

Comment:
What is this changed to?   Is this what discovery api is configured to point to?

Reply:
It's secret, can't tell you.

Reply:
![](https://media.giphy.com/media/3oFyCYNrra8qo1Cv8Q/giphy.gif)

Reply:
But in all seriousness I don't know.  I was referencing the smartcitiesdata wiki under Deploy guardian key. 

https://github.com/smartcitiesdata/smartcitiesdata/wiki/Example:-Setting-up-on-AWS

Reply:
The old value was the base64 encoded word "localhost" .  The new entry is a bunch of binary which does not properly base64 encode.  Unless we know what this value is and the app does not work without making this change, I would hesitate to make this change.

Reply:
$ cat | base64 -D
ce1RV/hBmrFVh11geYV4wPWvZ9vOF1en/tZTD/Qwlr27UmJka+Qi+jbFAZLu7FAu
q�QW�A��U�]`y�x��g��W���S�0���Rbdk�"�6����P

$ cat | base64 -D
bG9jYWxob3N0Cg==
localhost

Reply:
Made the changes back to local host